adobe-acrobat-sign-connector-product/README.md

```diff
@@ -52,6 +33,26 @@ The demo project can be used to test the authentication and signing and the demo
 
 ## Setup
 
+### Adobe Sign account creation
+
+  1. Create a **AdobeSign** Company **Account** OR for Developer Account creation use [Create Developer Account, APIs for custom applications | Acrobat Sign](https://www.adobe.com/sign/developer-form.html) and follow the steps:
+
+ 	 a. Fill form with personal and business data + continue
+ 	 ![fill-account-info](images/createAccountFillInfo.png)
+
+ 	 b. Provide a password + continue
+ 	 ![fill-password](images/createAccountPassword.png)
+
+ 	 c. Provide date of birth + continue
+ 	 ![fill-birth-date](images/createAccountBirthDate.png)
+
+ 	 d. You will receive a verification code. Insert the code. Will automatically continue
+ 	 ![verification-code](images/createAccountVerificationCode.png)
+
+ 	 e. Developer account is created
+ 	 ![account-finished](images/createAccountFinished.png)
+
+
 Adobe Sign provides 2 options for authentication. (See Setup section)
```

Comment:
this part ... until #Admin Setup Page ... is also somewhat asorted and doesn't help me. I think it should be under an own header? :thinking:  ... but I can't find a good name.

Reply:
also ok for me to leave it as is, maybe @ivy-sgi  or one of the original developers should re-work the full setup description :shrug: